libs/platform/src/lib/components/dynamic-page/dynamic-page.component.html

```diff
@@ -1,7 +1,7 @@
 <section class="fd-dynamic-page" [attr.aria-label]="ariaLabel">
-    <header class="fd-dynamic-page__header-fixed" #header>
+    <header class="fd-dynamic-page__header-fixed" [class]="{'fd-dynamic-page__header--not-collapsible': !headerCollapsible}" #header>
```

Comment:
`[class.fd-dynamic-page__header--not-collapsible]="!headerCollapsible"`

Reply:
ok done